src/PrestaShopBundle/Resources/config/services/adapter/tax_rules_group.yml

```
@@ -2,6 +2,22 @@ services:
   _defaults:
     public: true
 
+  prestashop.adapter.tax_rules_group.command_handler.add_tax_rules_group_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler\AddTaxRulesGroupHandler'
+    arguments:
+      - '@prestashop.adapter.tax_rules_group.repository.tax_rules_group_repository'
```

Same thing for the dependencies, this one is already migrated so you're lucky ^^
```suggestion
      - '@PrestaShop\PrestaShop\Adapter\TaxRulesGroup\Repository\TaxRulesGroupRepository'
```

---

But if you have some dependencies that are still only written with the name it's nice to migrate them (and add the alias in this case for retro compatibility) It's a best effort though not blocking for existing dependencies but a bit more for brand new ones

---

Fixed